### PR TITLE
Directly include parallel crypto3

### PR DESCRIPTION
**Pull request by AndreyMlashkin**

### CODE REVIEW

File: parallel-crypto3/parallel-crypto3.nix [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  propagatedBuildInputs = [ (if enableDebug then (enableDebugging boost) else boost) ];
+
+  buildInputs = [cmake_modules crypto3];
+
+  cmakeFlags =
+    [
+      (if runTests then "-DBUILD_TESTS=TRUE" else "")
+      (if runTests then "-DCMAKE_ENABLE_TESTS=TRUE" else "")
+      (if enableDebug then "-DCMAKE_BUILD_TYPE=Debug" else "-DCMAKE_BUILD_TYPE=Release")
+      (if enableDebug then "-DCMAKE_CXX_FLAGS=-ggdb" else "")
```
Comment (akokoshn) [inline parallel-crypto3/parallel-crypto3.nix:33]: again, it should be in cmake

File: parallel-crypto3/parallel-crypto3.nix [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{ lib,
+  stdenv,
+  ninja,
+  pkg-config,
+  cmake,
+  boost,
+  gdb,
+  cmake_modules,
```
Comment (akokoshn) [inline parallel-crypto3/parallel-crypto3.nix:8]: why we need it?

**Review by akokoshn — COMMENTED** [review]
let's make single one `.clang-format` for all repository

File: parallel-crypto3/parallel-crypto3.nix (hunk 2 above)
Comment (AndreyMlashkin) [reply parallel-crypto3/parallel-crypto3.nix:8]: Do you mean cmake_modules? it is used in parallel-crypto3 the same as in the original crypto3

File: parallel-crypto3/parallel-crypto3.nix (hunk 1 above)
Comment (AndreyMlashkin) [reply parallel-crypto3/parallel-crypto3.nix]: fixed

File: parallel-crypto3/parallel-crypto3.nix (hunk 2 above)
Comment (akokoshn) [reply parallel-crypto3/parallel-crypto3.nix:8]: got it, seem i can't to resolve threads

**Review by akokoshn — APPROVED** [review]
could you please remove empty `parallel-crypto3/.gitmodules`

**Pull request merged**